src/REPLICA/fix_pimd_langevin.h

```diff
@@ -0,0 +1,198 @@
+/* -*- c++ -*- ----------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   https://www.lammps.org/, Sandia National Laboratories
+   LAMMPS development team: developers@lammps.org
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+#ifdef FIX_CLASS
+// clang-format off
+FixStyle(pimd/langevin,FixPIMDLangevin);
+// clang-format on
+#else
+
+#ifndef FIX_PIMD_LANGEVIN_H
+#define FIX_PIMD_LANGEVIN_H
+
+#include "fix.h"
+
+namespace LAMMPS_NS {
+
+class FixPIMDLangevin : public Fix {
+ public:
+  FixPIMDLangevin(class LAMMPS *, int, char **);
+  ~FixPIMDLangevin() override;
+
+  int setmask() override;
+
+  void init() override;
+  void setup(int) override;
+//   void post_neighbor() override;
+  void post_force(int) override;
+  void initial_integrate(int) override;
+  void final_integrate() override;
+  void end_of_step() override;
+
+  double compute_vector(int) override;
+
+  /* System setting variables */
```

Comment:
Why are the properties and functions below public and not protected?

Reply:
The reason is that I learned the manner from fix_pimd.h when I started this work in late 2020. I have already pushed a commit that fixes this issue.